src/Domain/User/Entity/Maker.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\User\Entity;
+
+use App\Domain\User\UserInterface;
+use DateTimeImmutable;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Domain\User\Repository\MakerRepository")
+ */
+class Maker implements UserInterface
+{
+    /**
+     * @ORM\Column(type="guid")
+     * @ORM\Id
+     */
+    private string $id;
+
+    /**
+     * @ORM\Column(unique=true)
+     */
+    private string $email;
+
+    /**
+     * @var string[]
+     * @ORM\Column(type="json")
+     */
+    private array $roles = [];
+
+    /**
+     * @ORM\Column
+     */
+    private string $password;
+
+    /**
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private ?string $passwordResetToken = null;
+
+    /**
+     * @ORM\Column(type="datetime_immutable", nullable=true)
+     */
+    private ?DateTimeImmutable $passwordResetTokenCreatedAt;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $name;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $postalCode = null;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $addressCity = null;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $addressState = null;
+
+    /**
+     * @ORM\Column(name="latitude", type="decimal", precision=20, scale=16, nullable=true)
+     */
+    private ?string $latitude = null;
+
+    /**
+     * @ORM\Column(name="longitude", type="decimal", precision=20, scale=16, nullable=true)
+     */
+    private ?string $longitude = null;
+
+    public function __construct(string $email, string $name)
+    {
+        $this->email = $email;
+        $this->name = $name;
+        $this->id = Uuid::uuid4()->toString();
+    }
+
+    public function getId(): string
+    {
+        return $this->id;
+    }
+
+    public function getEmail(): string
+    {
+        return (string) $this->email;
+    }
+
+    public function getUsername(): string
+    {
+        return $this->email;
+    }
+
+    public function setEmail(string $email): self
+    {
+        $this->email = $email;
+
+        return $this;
```

Comment:
Except for a Builder, normal classes should not use a fluent interface. Here you can read why: https://ocramius.github.io/blog/fluent-interfaces-are-evil/

Reply:
Would you like to create an issue for this? :)

Reply:
Just remove it in this case. I´ll refactor the other code

Reply:
done #175